apps/meteor/client/views/account/security/ChangePassword.tsx

```diff
@@ -96,7 +114,7 @@ const ChangePassword = (props: AllHTMLAttributes<HTMLFormElement>) => {
 									disabled={!allowPasswordChange || !passwordIsValid}
 									aria-required={password !== '' ? 'true' : 'false'}
 									aria-invalid={errors.confirmationPassword ? 'true' : 'false'}
-									aria-describedby={`${confirmPasswordId}-error`}
+									aria-describedby={getAriaDescribedbyForPasswordVerification(!!errors.confirmationPassword)}
```

Comment:
WDYT of removing this function and just doing:

```suggestion
									aria-describedby={!!errors.confirmationPassword ? `${confirmPasswordId}-error` : undefined}
```